build/rlwrap/build.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/bash
+#
+# {{{ CDDL HEADER
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+# }}}
+
+# Copyright 2022 r7st r7st.guru@gmail.com
+
+. ../../lib/build.sh
+
+PROG=rlwrap
+VER=0.45.2
+PKG=ooce/util/rlwrap
+SUMMARY="readline wrapper"
+DESC="rlwrap is a 'readline wrapper', a small utility that uses the GNU Readline library to allow the editing of keyboard input for any command."
+
+set_arch 64
+
+set_mirror "$GITHUB/hanslub42/$PROG/releases/download"
+set_checksum sha256 "9f8870deb46e473d21b5db89d709b6497f4ef9fa06d44eebc5f821daa00c8eca"
+
+CFLAGS+=" -fstack-protector -gdwarf"
```

Comment:
You shouldn't need to set these explicitly, the framework takes care of adding the appropriate flags for stack protector and DWARF that is compatible with the CTF format, and varies depending on which omnios release is being built for. I just tried this on r151042 and see:

```
/opt/gcc-11/bin/gcc -DHAVE_CONFIG_H -I. -I..    -DDATADIR=\"/opt/ooce/share\" 
    -O2 -fno-aggressive-loop-optimizations
    -fstack-protector-strong -fstack-protector
    -gdwarf -gdwarf-2 -gstrict-dwarf
    -m64 -MT ptytty.o -MD -MP -MF .deps/ptytty.Tpo -c -o ptytty.o ptytty.c
```
so the framework is already adding `-fstack-protector-strong` and `-gdwarf-2 -gstrict-dwarf` and then converting the DWARF to the native CTF data. I think this line should just be deleted.